logstash_formatter.go

```diff
@@ -16,19 +16,15 @@ type LogstashFormatter struct {
 	// TimestampFormat sets the format used for timestamps.
 	TimestampFormat string
 
-	Hook *Hook
+	HookOnlyPrefix string
```

Comment:
I really don't want to add new fields to `LogstashFormatter`.
Lets remove it and see how to handle the prefix trimming in the `Fire` function.